torch_geometric/loader/utils.py

```diff
@@ -224,3 +224,21 @@ def filter_feature_store(
         data[attr.group_name][attr.attr_name] = tensors[i]
 
     return data
+
+
+def has_edge_index(data: Union[Data, HeteroData]) -> bool:
```

Comment:
Could this be defined on `Data`?

Reply:
Yeah we could add it to `HeteroData`, `Data` and probably `GraphStore`. That's much cleaner. But I doubt it would be used outside of `LinkNeighborLoader`.